ably/realtime/connection.py

```diff
@@ -167,6 +172,7 @@ async def ws_read_loop(self):
         while True:
             raw = await self.__websocket.recv()
             msg = json.loads(raw)
+            log.info(f'ws_read_loop receieved protocol message: {msg}')
```

Comment:
This didn't specify `ws_read_loop` is a function call like the rest

Reply:
fixed